service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseSubmissionValidatorServiceTest.java

```diff
@@ -0,0 +1,267 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.Children;
+import uk.gov.hmcts.reform.fpl.model.GroundsForEPO;
+import uk.gov.hmcts.reform.fpl.model.Hearing;
+import uk.gov.hmcts.reform.fpl.model.Orders;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+class CaseSubmissionValidatorServiceTest {
+
+    private final CaseSubmissionValidatorService service = new CaseSubmissionValidatorService();
+
+    @Test
+    void shouldReturnErrorsIfMandatoryCaseSectionsHaveNotBeenCompleted() {
+        CaseData caseData = CaseData.builder().build();
+        List<String> errors = service.validateCaseDetails(caseData);
+
+        assertThat(errors).contains("- Enter a case name");
```

Comment:
lets use .containsExactly or .containsOnly and pass in a list?

or we can do assertThat(errors).contains().contains()